near-sdk/src/lib.rs

```diff
@@ -369,6 +369,33 @@ extern crate quickcheck;
 /// }
 /// ```
 ///
+/// ## `#[deny_unknown_arguments]` (annotates methods of a type in its `impl` block)]
+///
+/// Specifies that the method call should error during deserialization if any unknown fields are present in the input.
+/// This helps ensure data integrity by rejecting potentially malformed input.
+///
+/// Without this attribute, unknown fields are silently ignored during deserialization.
+///
```

Comment:
```suggestion
/// 
/// Implementation uses [`deny_unknown_fields`](https://serde.rs/container-attrs.html#deny_unknown_fields) `serde`'s attribute.
/// 
/// In the following example call of `my_method` with
/// ```json,ignore
/// {
///     "description": "value of description"
/// }
/// ```
/// payload works, but call of `my_method` with
/// 
/// ```json,ignore
/// {
///     "description": "value of description",
///     "unknown_field": "what" 
/// }
/// ```
/// payload is declined with a `FunctionCallError(ExecutionError("Smart contract panicked: Failed to deserialize input from JSON."))` error.
/// 
```